README.md

```diff
@@ -6,20 +6,99 @@
 
 The official dojo 2 build command.
 
+*WARNING* This is beta software. It is not yet production ready, so you should use at your own risk.
```

Comment:
For consistency, please use the same phrasing found in say dojo/compose: **WARNING** This is *beta* software.  While we do not anticipate significant changes to the API at this stage, we may feel the need to do so.  This is not yet production ready, so you should use at your own risk.